testsuite/integration/microprofile-tck/config/pom.xml

```diff
@@ -173,7 +173,7 @@
                                             <name>standalone.xml</name>
                                             <layers>
                                                 <layer>datasources-web-server</layer>
-                                                <layer>observability</layer>
+                                                <layer>microprofile-config</layer>
```

Comment:
I'm not sure I want to do this as running against this combo is part of the test plan for our recommended combos in our images.

Reply:
The intention was to use a minimal profile sufficient to pass the MicroProfile Config TCK.  It also provides an opportunity to validate the microprofile-config layer.
https://github.com/wildfly/wildfly/pull/12774 will use a profile more in line with the test plan you described.
If you still disagree, I will revert this change.

Reply:
I'd prefer not to change this right now.  Right now #12774 is an empty module.

I don't particularly disagree with the change; I just don't feel like reopening a long discussion right now when I know there will be lots of other opportunities to do it. :)

Reply:
No problem.